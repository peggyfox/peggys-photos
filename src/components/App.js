import React from 'react';

// eslint-disable-next-line react/prefer-stateless-function
class App extends React.Component {
  render() {
    return (
      <div className="home">
        #peggysphotos
      </div>
    );
  }
}
export default App;
