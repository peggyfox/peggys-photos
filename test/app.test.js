import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/components/App';

describe('smoke test', () => {
  it('renders with no errors', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
