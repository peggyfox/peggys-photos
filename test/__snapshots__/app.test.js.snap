// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke test renders with no errors 1`] = `
<div
  className="container"
>
  <header>
    <h1>
      Peggys Photos
    </h1>
  </header>
  <div
    className="photos"
  >
    <div />
  </div>
  <div
    className="modal-container"
  />
</div>
`;
